### PR TITLE
Fix: Update documentation

### PR DESCRIPTION
**Pull request by caalador**
Updated documentation to match the
actual state of the project.
Removed image and replaced it with a ascii map
as the image is hard to update with all the constant
changes.

Fixes vaadin/flow#9595
Addresses documentation part of vaadin/flow#9627

### CODE REVIEW

**Comment by jouni** [comment]
…kicking the preview build, let’s see if it works now 🤞

**Review by jouni — APPROVED** [review]

File: articles/themes/application-theme.asciidoc [hunk 1]
```diff
@@ -13,7 +13,19 @@ Application Theme is the standard way to organize themes in a Vaadin application
...
+    ├── components                 | contains component specific themes
+    │   ├── vaadin-button.css      | component themes need to match the tag name
+    │   └── vaadin-text-field.css  |
+    ├── images                     | static resource folder
+    ├── webfonts                   | static resource folder
+    ├── document.css               | css always applied to the root of the page
+    ├── global.css                 | contains theme changes, may be targeted
+    ├── my-theme.generated.js      | auto-generated configuration file
+    └── theme.json                 | configuration for assets etc.
+```
```
Comment (jouni) [inline articles/themes/application-theme.asciidoc:28]:
```suggestion
[source, filesystem]
----
theme/                             | contains all application themes
└── my-theme/                      | contais files for 'my-theme'
    ├── components                 | contains component specific themes
    │   ├── vaadin-button.css      | component themes need to match the tag name
    │   └── vaadin-text-field.css  |
    ├── images                     | static resource folder
    ├── webfonts                   | static resource folder
    ├── document.css               | css always applied to the root of the page
    ├── global.css                 | contains theme changes, may be targeted
    ├── my-theme.generated.js      | auto-generated configuration file
    └── theme.json                 | configuration for assets etc.
----
```

There’s a [bug in the doc builder](https://github.com/vaadin/docs-app/issues/194), which makes this content disappear if the listing block language is not defined.

File: articles/themes/application-theme.asciidoc [hunk 2]
```diff
@@ -13,7 +13,19 @@ Application Theme is the standard way to organize themes in a Vaadin application
 Figure 1 illustrates a sample theme setup for "my-theme" application theme.
```
Comment (jouni) [inline articles/themes/application-theme.asciidoc:13]:
```suggestion
The following describes a sample file system structure for "my-theme".
```

**Review by jouni — CHANGES_REQUESTED** [review]

File: articles/themes/application-theme.asciidoc [hunk 3]
```diff
@@ -10,10 +10,23 @@ Application Theme is the standard way to organize themes in a Vaadin application
...
+    ├── components                 | contains component specific themes
+    │   ├── vaadin-button.css      | component themes need to match the tag name
+    │   └── vaadin-text-field.css  |
+    ├── images                     | static resource folder
+    ├── webfonts                   | static resource folder
+    ├── document.css               | css always applied to the root of the page
+    ├── global.css                 | contains theme changes, may be targeted
+    ├── my-theme.generated.js      | auto-generated configuration file
+    └── theme.json                 | configuration for assets etc.
+----
```
Comment (jouni) [inline articles/themes/application-theme.asciidoc:29]: Actually, this doesn’t work that great, since the text wraps on new lines, and makes it hard to follow.

We could try using callouts instead: https://asciidoctor.org/docs/user-manual/#callouts

Comment (jouni) [reply articles/themes/application-theme.asciidoc:29]: Another option is to use a table, with minimal styling (remove outer frame and inner grid lines): https://docs-app-preview.netlify.app/styling-test/tables/

Comment (caalador) [reply articles/themes/application-theme.asciidoc]: Callouts seem better than table in this instance. Updated.

**Review by pleku — APPROVED** [review]

File: articles/themes/application-theme.asciidoc (hunk 3 above)
Comment (jouni) [reply articles/themes/application-theme.asciidoc]: Thanks. Just one problem: callouts are currently broken: https://github.com/vaadin/docs-app/issues/179

But we’ll get those fixed.

Comment (pleku) [reply articles/themes/application-theme.asciidoc]: I saw that too from the preview, but I guess it is fine for now.
